source/whats-new.txt

```diff
@@ -122,6 +122,13 @@ The 3.2 driver release includes the following new features:
   To learn more about Atlas Vector Search with the {+driver-short+}, see the
   :ref:`csharp-atlas-vector-search` guide.
 
+- Adds the ``DocumentFormat`` property to the ``DateOnlySerializer``.
+  This property allows you to customize the way that the driver stores
+  ``DateOnly`` values in nested documents. This release also adds the
```

Comment:
I think just saying "values" as it was before make more sense. Now it seems like it's influencing only DateOnly values in subdocuments.